docs/distributions/gentoo/installation.md

```diff
@@ -2,10 +2,11 @@
 
 ## Hardware Requirements
 
+* Gentoo Minimal Installation ISO
+    * Requirements for this ISO: USB keyboard, mouse, ethernet adapter/Wi-Fi adapter
+    * If you don't meet the requirements, then you are required to use another distro's ISO. A recommended distro to use is Ubuntu.
```

Comment:
Can you clarify that they want mbp-ubuntu, so they don't use a normal ubuntu iso?